examples/performance-debugging/doc/results.dox

```diff
@@ -0,0 +1,175 @@
+<h1>Results</h1>
+This is the expected output:
+
+@code{.cpp}
```

Comment:
I think you should also add the output you write to the file here as well as that is probably what people will be looking for.